nemo_curator/datasets/image_text_pair_dataset.py

```diff
@@ -79,7 +79,11 @@ def from_webdataset(cls, path: str, id_col: str):
             path (str): The path to the WebDataset-like format on disk or cloud storage.
             id_col (str): The column storing the unique identifier for each record.
         """
-        metadata = dask_cudf.read_parquet(path)
+        metadata = dask_cudf.read_parquet(path, split_row_groups=False)
+        # TODO: This is a hack to ensure that the number of partitions is not combined
+        # and remain the same as the number of shards.
+        # DEBUG: Why is this happening?
+        metadata = metadata.repartition(npartitions=metadata.npartitions)
```

Comment:
Curious why was this an issue / where was it caught?

Reply:
Let me file an issue for this to dask_cudf/dask (will merge after a file an issue and link it here) .